### ISSUE TITLE
devskim --ignore-rules-ids doesn't work

### ISSUE THREAD

**Issue by daltonv** [oxsecurity/megalinter#2780]
**Describe the bug**
When adding the --ignore-rules-ids argument to devskim it breaks without giving a descriptive error

**To Reproduce**

Use the v7.1.0 megalinter release.
Use the standard `mega-linter.yml` github actions workflow

Use the following `.mega-linter.yml` config:
```yaml
---
# Configuration file for MegaLinter
# See all available variables at https://megalinter.io/configuration/ and in linters documentation

APPLY_FIXES: none # all, none, or list of linter keys
# ENABLE: # If you use ENABLE variable, all other languages/formats/tooling-formats will be disabled by default
# ENABLE_LINTERS: # If you use ENABLE_LINTERS variable, all other linters will be disabled by default
DISABLE:
  - COPYPASTE # Comment to enable checks of excessive copy-pastes
  # - SPELL # Uncomment to disable checks of spelling mistakes
SHOW_ELAPSED_TIME: true
FILEIO_REPORTER: false
# DISABLE_ERRORS: true # Uncomment if you want MegaLinter to detect errors but not block CI to pass
FILTER_REGEX_EXCLUDE: (qac/prqa/)
PRE_COMMANDS:
  - command: pip install conan invoke
    venv: pylint
REPOSITORY_DEVSKIM_ARGUMENTS: ['--ignore-rules-ids','DS121708']
```

Megalinter will now produce an error for the devskim job

**Expected behavior**
I would expect the `--ignore-rules-ids` argument to work without error.

**Screenshots**
Here is the output of megalinter with the not so helpful error

```
❌ Linted [REPOSITORY] files with [devskim]: Found 1 error(s) - (0.09s) (expand for details)
  - Using [devskim v0.7.104] https://megalinter.io/7.1.0/descriptors/repository_devskim
  - MegaLinter key: [REPOSITORY_DEVSKIM]
  - Rules config: identified by [devskim]
  --Error detail:
```

**Additional context**
This might be related to another devskim issue I saw filed https://github.com/oxsecurity/megalinter/issues/1993

**Comment by daltonv** [comment]
I also just checked to see if changing the megalinter action to the latest pre release version with
```yaml
uses: oxsecurity/megalinter@main
```
fixes this, but I still see the same problem.

**Comment by nvuillam** [comment]
Please can you try with `LOG_LEVEL: DEBUG` in your .mega-linter.yml config file so we can see the exact CLI call ?

**Comment by daltonv** [comment]
@nvuillam no problem. Here is the relevant output with the debug enabled
```console
[devskim] command: ['devskim', 'analyze', '--ignore-rules-ids', 'DS121708', '.', '-E', '--ignore-globs', '**/megalinter-reports/**']
[devskim] CWD: /github/workspace
[devskim] result: 2 
Unable to get number of errors with regex_count and (\[.*\])
Linter version command: ['/root/.dotnet/tools/devskim', '--version']
Linter version result: 0 Microsoft DevSkim Command Line Interface
0.7.104+bd34d6c82b

❌ Linted [REPOSITORY] files with [devskim]: Found 1 error(s) - (0.11s) (expand for details)
  - Using [devskim v0.7.104] https://megalinter.io/7.1.0/descriptors/repository_devskim
  - MegaLinter key: [REPOSITORY_DEVSKIM]
  - Rules config: identified by [devskim]
  --Error detail:
```

**Comment by daltonv** [comment]
Ah I made some progress. I figured out how to install devskim v0.7.104 locally on ubuntu and ran the exact same command as CI reports:
```bash
devskim analyze --ignore-rule-ids DS121708 . -E --ignore-globs **/megalinter-reports/**
```
However this also prints no error, but I see the RC is 2
```bash
$ echo $?
2
```

If I move the --ignore-rule-ids like so
```bash
devskim analyze . --ignore-rule-ids DS121708  -E --ignore-globs **/megalinter-reports/**
```
It works, but the rule is not ignored

Turns out this cli option does not exist in the megalinter version of devskim. At least that's what it looks like from the `devskim analyze --help` command. 

# TLDR 
What I really need is for megalinter to update to the latest devskim release, as errors are actually reported on stdout/stderr and the --ignore-rule-ids command exists.
